### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN005.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#51] [General]
# https://blog.3place.cn/3place/lmyn/LMYN005.html

https://blog.3place.cn/3place/lmyn/LMYN005.html

**Comment by picklecai** [comment]
蚁垤接着昨天的6步：
7. 完成作品
8. 传布作品

从1-8，这是一个完整的周期。
6、7是耗精力最多的。
一般的作者，好像并不考虑步骤8。除非官方需要。

所以，蚁垤，或者之类的编纂者，可能是官方背景。帝王需要这样一个作品来教化百姓。所以特意完美化主人公，所以最后还要考虑怎样传播到人群中去。

**Comment by picklecai** [comment]
走进议事堂，
跳出生死轮。

走进议事堂：精神成仙，肉体成王？
